Packs/APIVoid/Integrations/APIVoid/APIVoid.py

```diff
@@ -0,0 +1,621 @@
+
+    ip = args.get('ip')
+    raw_response = client.check_ip(ip)
+
+    if 'error' in raw_response:
+        return_error("Command Failed: " + str(raw_response['error']))
+
+    report = raw_response.get('data', {}).get('report', None)
+    if report:
+        engines = report.get('blacklists', {}).get('engines_count', 0)
+        detections = report.get('blacklists', {}).get('detections', 0)
+
+        # IP Information
+        information = report.get('information', {})
+        lat = information.get('latitude', None)
+        lng = information.get('longitude', None)
+        ip = {
+            'Address': report['ip'],
+            'Hostname': information.get('reverse_dns', None),
+            'Geo': {
+                'Location': f'{lat}:{lng}' if lat and lng else None,
+                'Country': information.get('country_name', None),
+                'Description': information.get('isp', None),
+            },
+            'DetectionEngines': engines,
+            'PositiveDetections': detections,
+        }
+        ec = indicator_context(client, ip, outputPaths['ip'], 'Address', engines, detections)
+
+        if not reputation_only:
+            ec['APIVoid.IP(val.ip && val.ip == obj.ip)'] = report
+
+        md = tableToMarkdown(f'APIVoid information for {ip["Address"]}:', ip)
+
+    else:
+        ec = {}
+        md = f'## No information for {ip}'
+
+    return_outputs(md, ec, raw_response)
+
+
+def domain_command(client, args, reputation_only):
+
+    domain = args.get('domain')
+    raw_response = client.check_domain(domain)
+
+    if 'error' in raw_response:
+        return_error("Command Failed: " + str(raw_response['error']))
+
+    report = raw_response.get('data', {}).get('report', None)
+    if report:
+        engines = report.get('blacklists', {}).get('engines_count', 0)
+        detections = report.get('blacklists', {}).get('detections', 0)
+
+        # Domain Information
+        domain = {
+            'Name': report['host'],
+            'DNS': report['host'],
+            'DetectionEngines': engines,
+            'PositiveDetections': detections,
+        }
+
+        ec = indicator_context(client, domain, outputPaths['domain'], 'Name', engines, detections)
+        md = tableToMarkdown(f'APIVoid information for {domain["Name"]}:', domain)
+
+        if not reputation_only:
+            ec['APIVoid.Domain(val.domain && val.domain == obj.domain)'] = report
+
+    else:
+        ec = {}
+        md = f'## No information for {domain}'
+
+    return_outputs(md, ec, raw_response)
+
+
+def url_command(client, args, reputation_only):
+
+    url = args.get('url')
+    raw_response = client.check_url(url)
+
+    if 'error' in raw_response:
+        return_error("Command Failed: " + str(raw_response['error']))
+
+    report = raw_response.get('data', {}).get('report', None)
+    if report:
+        report['url'] = url
+
+        engines = len(report.get('domain_blacklist', {}).get('engines', 0))
```

Comment:
what type is it?
list or number? 
in the IP one it looks different

Reply:
the default value at the end should be an empty list.
will fix